src/wallet/interfaces.cpp

```diff
@@ -255,15 +255,13 @@ class WalletImpl : public Wallet
         LOCK(m_wallet->cs_wallet);
         return m_wallet->IsLockedCoin(output.hash, output.n);
     }
-    void listLockedCoins(std::vector<COutPoint>& outputs) override
+    std::vector<COutPoint> listLockedCoins() override
     {
-        LOCK(m_wallet->cs_wallet);
-        return m_wallet->ListLockedCoins(outputs);
+        return WITH_LOCK(m_wallet->cs_wallet, return m_wallet->ListLockedCoins());
```

Comment:
nit: please don't do this in the future. No need to use WITH_LOCK here, plus this makes the diff larger for no benefit

Reply:
Resolved in latest push